src/sections/Resources/Resources-grid/filters.js

```diff
@@ -14,31 +14,31 @@ const Navigation = (props) => {
     graphql`
             query allFilters {
                 type: allMdx(
-                    filter: { fields: { collection: { in: ["blog", "resources", "news", "events"] } }, frontmatter: { published: { eq: true } , resource: { eq: true} }}
+                    filter: { fields: { collection: { in: ["blog", "resources", "news", "events", "tutorials"] } }, frontmatter: { published: { eq: true } , resource: { eq: true} }}
```

Comment:
```suggestion
                    filter: { fields: { collection: { in: ["blog", "resources", "news", "events"] } }, frontmatter: { published: { eq: true } , resource: { eq: true} }}
```
This won't be required anymore.